e2e/contrast-uploadPaper.spec.ts

```diff
@@ -147,3 +108,42 @@ test("user can add an experiment", async ({ page }) => {
       .isVisible()
   );
 });
+
+test("user can delete paper from Uncompleted submissions", async ({ page }) => {
+  await login({ page });
+  await page.getByRole("link", { name: "Upload New Paper" }).click();
+  await page.getByRole("link", { name: "Upload New Paper" }).click();
+  await page.getByText("Uncompleted submissions(1)").click();
+  await page.getByText("delete").click();
+  await page.getByRole("button", { name: "Yes' Delete it!" }).click();
+  expect(await page.getByText("Study was deleted successfully").isVisible());
+});
+
+test("user cant add new paper if not all fields r full", async ({ page }) => {
+  await login({ page });
+  await page.getByRole("link", { name: "Upload New Paper" }).click();
+  await page.getByRole("link", { name: "Upload New Paper" }).click();
+
+  await page.getByRole("button", { name: "Add New Paper" }).click();
+  await page.getByPlaceholder("Enter New Paper Title").click();
+  await page.getByPlaceholder("Enter New Paper Title").fill("TestPaper");
+  await page.getByPlaceholder("Enter your DOI identifier").click();
+  await page.getByPlaceholder("Enter your DOI identifier").click();
+  await page
+    .getByPlaceholder("Enter your DOI identifier")
+    .fill("10.1177/00225266209743");
+  //skip year field
+  await page
+    .locator("div")
+    .filter({ hasText: /^Select or Add Authors$/ })
+    .nth(2)
```

Comment:
Of you have a choice better not use the nth because its very falky

Reply:
יש לך דרך אחרת להציע? זה כדי לבחור את האפשרות השנייה/שלישית (עד כמה שאני מבין)

Reply:
תראה הכי טוב לדעתי זה לבחור:
או את האפשרות לפי הטקסט שלה
או אם האפשרות בעייתי, אפשר לשים data=testid על ההחלק של הקופוננטה\ טאג, או להשתמש ב id. 

למה?
כי אם מחר יתתווסף לך עוד אפשרות, בהתחלה, אז היא כבר לא תהיה השנייה אלא הרביעית וכו'